File: java/app/src/main/java/net/fbvictorhugo/j/barreirasanitaria/data/dao/QuestionarioDAO.java
package net.fbvictorhugo.j.barreirasanitaria.data.dao;

import net.fbvictorhugo.j.barreirasanitaria.data.model.Questionario;

import java.util.List;

public interface QuestionarioDAO {

    List<Questionario> listar();

    void inserir(Questionario questionario) throws Exception;

}

File: kotlin/app/src/main/java/net/fbvictorhugo/k/barreirasanitaria/data/dao/QuestionarioDAO.kt
package net.fbvictorhugo.k.barreirasanitaria.data.dao

import net.fbvictorhugo.k.barreirasanitaria.data.model.Questionario

interface QuestionarioDAO {
    fun listar(): ArrayList<Questionario>

    @Throws(Exception::class)
    fun inserir(questionario: Questionario)

}